backend/routes/api/reviews.js

```diff
@@ -155,4 +161,38 @@ router.delete('/:id', requireAuth, async (req, res, next) => {
     });
 });
 
+//delete a review image
+router.delete('/:id/images/:imageId', requireAuth, async (req, res, next) => {
+    const userId = req.user.id;
+    const reviewId = +req.params.id;
+    const imageId = +req.params.imageId;
+    const reviewByID = await Review.findByPk(reviewId);
+
+    if(!reviewByID){
+        const err = new Error("Review couldn't be found");
+        err.status = 404;
+        return next(err);
+    }
+    if(userId !== reviewByID.userId){
+        const err = new Error("Forbidden");
+        err.status = 403;
+        return next(err);
+    }
+
```

Comment:
May have a work around in deleting an image of a review that doesn't belong to current user

example : 
- review queried for by the id provided by the params may belong to the current user
- image queried for by the imageId provided by the params isnt associated with the review queried for ( reviewId on the image record doesn't match the review id provided

Reply:
@E-F-III added where clause to grab the image belongs to the review